bitnami/rabbitmq/values.yaml

```diff
@@ -340,6 +340,8 @@ configuration: |-
 extraConfiguration: |-
   #default_vhost = {{ .Release.Namespace }}-vhost
   #disk_free_limit.absolute = 50MB
```

Comment:
you can remove this commented line (since you're defining the same value below)

Reply:
I like to keep to reduce to the max the nb of line changed compared to master

Reply:
ha yes, ok